src/Path/Scope.hs

```diff
@@ -1,47 +1,168 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving, TypeOperators #-}
+{-# LANGUAGE DeriveGeneric, DeriveTraversable, FlexibleInstances, LambdaCase, MultiParamTypeClasses, QuantifiedConstraints, RankNTypes, StandaloneDeriving #-}
 module Path.Scope where
```

Comment:
The old `Scope` datatype was briefly renamed to `Namespace` before being deleted; this module is entirely new, offering scope-safe syntax helpers.